chainer/training/extensions/__init__.py

```diff
@@ -0,0 +1,19 @@
+from chainer.training.extensions import _snapshot
+from chainer.training.extensions import computational_graph
+from chainer.training.extensions import evaluator
+from chainer.training.extensions import exponential_decay
+from chainer.training.extensions import linear_shift
+from chainer.training.extensions import log_report
+from chainer.training.extensions import print_report
+from chainer.training.extensions import progress_bar
+
+
+dump_graph = computational_graph.dump_graph
+Evaluator = evaluator.Evaluator
+ExponentialDecay = exponential_decay.ExponentialDecay
+LinearShift = linear_shift.LinearShift
+LogReport = log_report.LogReport
+snapshot = _snapshot.snapshot
+snapshot_object = _snapshot.snapshot_object
```

Comment:
This is a problem of symbol conflicts... Maybe I should think of a better name. Do you have a good idea?